app/models/agents/user_location_agent.rb

```diff
@@ -1,4 +1,5 @@
 require 'securerandom'
+require 'haversine'
```

Comment:
If you're going to add a dependency, it would be great if you could implement `gem_dependency_check`, which we use to find out if an Agent has all the dependencies it needs to run.  This allows people who don't use the Agent to disable the dependencies to save RAM.

See https://github.com/cantino/huginn/blob/master/app/models/agents/ftpsite_agent.rb#L9 and https://github.com/cantino/huginn/blob/master/app/models/agents/ftpsite_agent.rb#L12 for an example of how to declare the dependencies and check for them to make a helpful description.